tests/framework/e2e/etcd_process.go

```diff
@@ -179,3 +182,14 @@ func (ep *EtcdServerProcess) Logs() LogsExpect {
 	}
 	return ep.proc
 }
+
+func ExpectLog(t *testing.T, ep EtcdProcess, expectLog string) {
```

Comment:
Don't like this helper function because:
* It has confusing naming `Expect` doesn't imply calling `t.Fatal` in context of e2e tests as it's used differently throughout code
* It's so short that it adds code instead of shortening it.

Reply:
I have rename this function as AssertProcessLogs but happy to name it something that fits more with the projects naming conventions

As an argument to keeping it as a helper, it already has 4 callers in 2 different files, and in my personal experience, waiting for a log message is a sadly common way of making end-to-end tests